### PR TITLE
added partial and did more styling

### PR DESCRIPTION
**Pull request by AbdullahAlger**
Added a form partial and did some styling.

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -66,6 +66,8 @@ body {
       }
     }
 
+@import 'home';
+@import 'wikis';
```
Comment (eliotsykes) [inline app/assets/stylesheets/application.scss]: Consider making `application.scss` only having `@import` statements, and move all CSS rule sets to `@import`-ed files. I've found it tends to make styles a bit easier to follow and maintain if application.scss is organized in this way.

File: app/views/wikis/edit.html.erb [hunk 2]
```diff
@@ -1,19 +1,6 @@
...
-      </div>
-      <div class="form-group">
-        <%= f.label "Edit Wiki" %>
-        <%= f.text_area :body, rows: 20, class: "form-control", placeholder: "Enter wiki here ..." %>
-      </div>
-      <div class="form-group">
-        <%= f.submit "Save", class: "btn btn-success" %>
-      </div>
-      <% end %>
+    <%= render partial: "form", locals: { wiki: @wiki } %>
```
Comment (eliotsykes) [inline app/views/wikis/edit.html.erb]: Good use of partial template for wiki form.

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (AbdullahAlger) [inline app/assets/stylesheets/application.scss]: I've done that using Angular, but I didn't know if it was wise to put your index page styles inside of other files and use only imports. I guess I should have figured that out.

File: app/views/wikis/edit.html.erb (hunk 2 above)
Comment (AbdullahAlger) [inline app/views/wikis/edit.html.erb]: I was wondering how to use render by only using 
`<%= render wiki %> 
I don't know if this is possible. I tried it, but got an error.

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (AbdullahAlger) [inline app/assets/stylesheets/application.scss]: I mean creating Angular front-end projects.

File: app/views/wikis/index.html.erb [hunk 3]
```diff
@@ -1,13 +1,19 @@
...
     <h1>All Wikis</h1>
-    <div class="col-xs-8">
+    <div class="col-xs-6">
       <% @wikis.each do |wiki| %>
         <div class="title">
-          <%= link_to wiki.title, wiki %>
+          <span class="title-top"><%= link_to wiki.title, wiki %></span>
+          <span>
+            <small>
+              Created by: <%= wiki.user.name %>, <%= time_ago_in_words(wiki.created_at) %> ago
```
Comment (eliotsykes) [inline app/views/wikis/index.html.erb]: :+1: for use of a Rails' built-in helper method here, nicely done.

If you've not seen these yet, review the various helper methods Rails provides:
- http://guides.rubyonrails.org/action_view_overview.html#overview-of-helpers-provided-by-action-view
- http://api.rubyonrails.org/classes/ActionView/Helpers.html

File: app/views/wikis/edit.html.erb (hunk 2 above)
Comment (eliotsykes) [inline app/views/wikis/edit.html.erb]: The error might hint at what the path the partial template needs to be. You can find out more about how this shorthand works from this section of the API docs:

Look for "Rendering the default case" section at http://api.rubyonrails.org/classes/ActionView/PartialRenderer.html

*However* I prefer the way you've got it now as this is for a form. If it was for a read-only (show) view and for a form, then maybe use the shorthand, as that's what it tends to get used for. The shorthand isn't used for forms so much.

Comment (eliotsykes) [inline app/views/wikis/edit.html.erb]: @AbdullahAlger I updated the last paragraph in the note above this one, it was missing the **not** (in bold) that I've just added.

**Pull request merged**